src/version.rs

```diff
@@ -54,7 +54,8 @@ use std::fmt::Write;
 /// Details: [`const char *GDALVersionInfo(const char*)`](https://gdal.org/api/raster_c_api.html#_CPPv415GDALVersionInfoPKc)
 pub fn version_info(key: &str) -> String {
     let c_key = CString::new(key.as_bytes()).unwrap();
-    _string(unsafe { gdal_sys::GDALVersionInfo(c_key.as_ptr()) })
+    let rv = unsafe { gdal_sys::GDALVersionInfo(c_key.as_ptr()) };
```

Comment:
Do we need to free `rv` here? I assume that this is the equivalent of a `&'static str` on the c side, right?

Reply:
I think it's more complicated (it's a string that GDAL formats and stores in TLS IIRC), but according to the docs:

> an internal string containing the requested information

So we can treat it as static.

Reply:
> Do we need to free `rv` here?

As a rule of thumb, that part of the GDAL C API should be relatively easy to get:
- For C/C++ functions that return a "const char*", as a "const char*" cannot be passed directly to C free(),  that indicates that the returned value should not be freed
- And the documentation of functions where the return value must be freed indicates so, and generally will specify which free function is expected. (like OGR_G_DestroyGeometry() for anything OGRGeometryH related, etc)